pkger.go

```diff
@@ -25,13 +25,13 @@ var disk = func() pkging.Pkger {
 		log.Println(err)
 	}
 	return n
-}()
+}
 
 func impl() pkging.Pkger {
 	gil.RLock()
 	defer gil.RUnlock()
 	if current == nil {
-		return disk
+		return disk()
```

Comment:
this will create a brand new disk for every call, that's not good.

Reply:
So we have to memoize the result of `disk()` ?

Reply:
yes, that's what the code was doing.

Reply:
Would this be ok ?
```suggestion
		current = disk()
```

Reply:
No, it will radically change Pkger's behavior. lol. 

Pkger wraps `pkging.Pkger` implementations when using `pkger.Apply`. this allows for layering implementations, much like `context.Context`. 

But setting `current` like that, you will get 2 `pkging.Pkger` implementations in the binary, a `mem.Pkger` that falls back to `stdos.Pkger`. So it will always look to the file system, which isn't correct.

Reply:
Those errors appear, at least by the error messages, that modules aren't being used. Pkger requires modules, so perhaps that's the issue?

Reply:
What is the expected behavior?

Is it a `mem.Pkger` which fallbacks to nothing?

That's what I would expect in my case, But I'm not sure if it is the general expected behavior...

Reply:
Correct.

Reply:
fun story, despite knowing all of that, I made the exact same commit at one point and broke everything. lol.